### ISSUE TITLE
Unexpected error. Error: EEXIST: file already exists in using community-template sec-insights

### ISSUE THREAD

**Issue by farazfarid** [run-llama/create-llama#31]
Hey there I am trying to set up a project using the sec template even tho the folder was completely empty when I did as project name `./` or create a new project altogether the files were created but I got this error exception:

```
npx create-llama@latest
√ What is your project named? ... ./
√ Which template would you like to use? » Community template from https://github.com/run-llama/create_llama_projects
√ Select community template » sec-insights
Creating a new LlamaIndex app in C:\Users\test\Documents\Coding\llamaindex-bastion.


Installing community project: sec-insights
Adding .devcontainer

Aborting installation.
Unexpected error. Please report it as a bug:
 Error: EEXIST: file already exists, mkdir 'C:\Users\test\Documents\Coding\llamaindex-bastion\.devcontainer'
    at Object.mkdirSync (node:fs:1380:26)
    at writeDevcontainer (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:17421)
    at createApp (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:19819)
    at async run (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:62870) {
  errno: -4075,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: 'C:\\Users\\test\\Documents\\Coding\\llamaindex-bastion\\.devcontainer'
}
```

```
npx create-llama@latest
√ What is your project named? ... llamaindex-bastion
√ Which template would you like to use? » Community template from https://github.com/run-llama/create_llama_projects
√ Select community template » sec-insights
Creating a new LlamaIndex app in C:\Users\test\Documents\Coding\llamaindex-bastion.


Installing community project: sec-insights
Adding .devcontainer

Aborting installation.
Unexpected error. Please report it as a bug:
 Error: EEXIST: file already exists, mkdir 'C:\Users\test\Documents\Coding\llamaindex-bastion\.devcontainer'
    at Object.mkdirSync (node:fs:1380:26)
    at writeDevcontainer (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:17421)
    at createApp (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:19819)
    at async run (C:\Users\test\AppData\Local\npm-cache\_npx\7bfc2205dda2d438\node_modules\create-llama\dist\index.js:298:62870) {
  errno: -4075,
  code: 'EEXIST',
  syscall: 'mkdir',
  path: 'C:\\Users\\test\\Documents\\Coding\\llamaindex-bastion\\.devcontainer'
}
```

**Comment by marcusschiesser** [comment]
Thanks @farazfarid. This is fixed in the latest 0.0.32 release

**Comment by farazfarid** [comment]
Thankyou @marcusschiesser that sounds amazing I can't wait to implement it!
